src/style.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
+  font-family: "avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.big_searchbar {
+  margin: 20px auto;
+}
+
+.search {
+  width: 100%;
+  padding: 23px 62px;
+  box-sizing: border-box;
+  background: url("images/Search.svg") no-repeat 25px center;
+  font-family: "avenir", helvetica, sans-serif;
+  font-weight: 700px;
+  font-size: 16px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.search_small {
+  padding: 10px 33px;
+  background-position: 12px 14px;
+  background-size: 11px;
+  font-size: 14px;
+}
+
+.search::placeholder {
+  font-family: "avenir", helvetica, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
```

Comment:
Put this color in the main class, it should be like that not only for placeholder